client/layout/masterbar/style.scss

```diff
@@ -3,9 +3,14 @@ $autobar-height: 20px;
 
 // The WordPress.com Masterbar
 .masterbar {
-	background: $masterbar-color;
-	border-bottom: 1px solid darken( $masterbar-color, 4% );
-	color: $white;
+	border-bottom: 1px solid;
+	@include theme-multiple(
+		(
+			'background': --masterbar-bg-color,
+			'border-color': --masterbar-border-color,
+			'color': --masterbar-color,
+		)
+	);
```

Comment:
Yep, I think it's better to use the latter, as it would avoid learning and understanding two syntaxes that do the same, but also importantly would keep the indentation in the SCSS files uniquely for the class hierarchy.